pkg/k8s/network_policy_test.go

```diff
@@ -152,6 +152,48 @@ func (s *K8sSuite) TestParseNetworkPolicy(c *C) {
 	c.Assert(repo.CanReachRLocked(&ctx), Not(Equals), api.Allowed)
 }
 
+func (s *K8sSuite) TestParseNetworkPolicyNoSelectors(c *C) {
+
+	// Ingress with neither pod nor namespace selector set.
+	ex1 := []byte(`{
+"kind": "NetworkPolicy",
```

Comment:
Minor nitpick - since it's json you can indent this string, but it's nothing of great importance.